server/jetstream_cluster.go

```diff
@@ -83,6 +83,14 @@ type Placement struct {
 	Tags    []string `json:"tags,omitempty"`
 }
 
+// Used for stepdown requests to guide placement in clustered JetStream. Separate
+// struct because we don't want PreferredServer field to be part of the API in
+// stream configs etc.
```

Comment:
We could allow this in normal Placement. And we could honor it if selected when creating a stream IMO.

Reply:
Yep, this PR just deals with the metalayer but could do for streams/consumers for sure.